docs/sources/panels/transformations/remove.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Remove Transformations"
```

Comment:
I would not make this a separate topic. Too short.

Reply:
Agree